app/src/main/java/com/example/pix/chat/fragments/ChatFragment.java

```diff
@@ -100,11 +101,17 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
         ivPictures = view.findViewById(R.id.chat_pictures);
         ivNewPic = view.findViewById(R.id.chat_image);
 
+        ComposeFragment composeFragment = new ComposeFragment();
+        composeFragment.setSharedElementEnterTransition(new Explode());
+        composeFragment.setEnterTransition(new Explode());
+        setExitTransition(new Explode());
+        composeFragment.setSharedElementReturnTransition(new Explode());
+
         // If we click the camera, go to the ComposeFragment
         ivCamera.setOnClickListener(view1 -> {
             getActivity().getSupportFragmentManager()
                     .beginTransaction()
-                    .replace(R.id.friend_container, new ComposeFragment())
+                    .replace(R.id.friend_container, composeFragment)
```

Comment:
I want to use the same ComposeFragment for this friend to improve transition speed.